CSS/mobilestyle.css

```diff
@@ -331,6 +332,44 @@ body {
   overflow-y: scroll;
 }
 
+/* .popupPage {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: rgba(0, 15, 24, 0.6);
+  backdrop-filter: blur(28px);
+  width: 96%;
+  height: 100vh;
+  width: 100%;
+  z-index: 10;
+  padding: 2%;
+  overflow-y: scroll;
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  margin: 5%;
+  position: relative;
+  background-color: #fff;
+  z-index: 1;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  height: 100vh;
+  width: 94%;
+  box-sizing: border-box;
+  border: 1px solid #ebecf0;
+  padding: 4%;
+  border-radius: 6px;
+} */
```

Comment:
- [ ] These commented lines of code should be removed please, it makes your code unclean and not professional at all